### PR TITLE
RDM-9053: Postgress 11 upgrade

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
RDM-9053: Postgress 11 upgrade
    
           [RDM-9053] (https://tools.hmcts.net/jira/browse/RDM-9053).

### CODE REVIEW

**Review by mario-paniccia — APPROVED** [review]
👍

File: infrastructure/main.tf [hunk 1]
```diff
@@ -93,3 +93,57 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
   value        = module.definition-store-db.postgresql_database
   key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
 }
+
+////////////////////////////////
```
Comment (Thor-tech-of-metal) [inline infrastructure/main.tf:97]: This code should have already been merged in develop

File: charts/ccd-definition-store-api/values.demo.template.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    ccd:
+      secrets:
+        - definition-store-api-POSTGRES-PASS-V11
+        - ccd-definition-s2s-secret
+        - ccd-ELASTIC-SEARCH-URL
+        - ccd-ELASTIC-SEARCH-PASSWORD
+        - storage-account-primary-connection-string
+        - AppInsightsInstrumentationKey
+  environment:
+    FLYWAY_NOOP_STRATEGY: true
```
Comment (mario-paniccia) [inline charts/ccd-definition-store-api/values.demo.template.yaml:12]: `FLYWAY_NOOP_STRATEGY: true`
do we need this? it's already in `values.yml`

Comment (smathangi) [reply charts/ccd-definition-store-api/values.demo.template.yaml:12]: Demo deployment is by flux. You need to add to flux (demo.yaml)

**Pull request closed without merging**